src/components/registration/FormFieldGroup.tsx

```diff
@@ -49,7 +49,12 @@ const FormFieldGroup = ({ testId, fields, onChange }: FormFieldGroupProps) => {
             }}
           >
             {field.question_type === QuestionTypes.MULTIPLE_CHOICE && (
-              <MenuItem value="none">None</MenuItem>
+              <>
+                <MenuItem value="">None</MenuItem>
+                {field.options.map((option: string) => (
+                  <MenuItem value={option}>{option}</MenuItem>
+                ))}
+              </>
```

Comment:
Is this a smelly use of fragments @brittanylau?

Reply:
nope this is a great use case :)

Reply:
![image](https://user-images.githubusercontent.com/21024100/121718123-eed5ab00-caaf-11eb-9f4d-7cbe8ccc5413.png) ur right ... I gotta stop testing my code after midnight